frontend/api/members.graphql

```diff
@@ -12,6 +12,9 @@ query MeHeader {
         id
         name
         nameEn
+        committee {
+          name
+        }
```

Comment:
Do we need to fetch this in the Me-query?

Reply:
No, you are correct. Removing.